mlrun/db/httpdb.py

```diff
@@ -98,6 +99,12 @@ class HTTPRunDB(RunDBInterface):
     """
 
     kind = "http"
+    # by default we don't retry on POST request as they are usually not idempotent
+    # here is a list of identified POST requests that are idempotent ('store' vs 'create') and can be retried
+    RETRIABLE_POST_PATHS = [
+        r"\/?projects\/.*\/artifacts\/.*\/.*",
+        r"\/?run\/.*\/.*",
```

Comment:
`.*` means 0 or more chars, `.+` is one or more.
```suggestion
        r"\/?projects\/.+\/artifacts\/.+\/.+",
        r"\/?run\/.+\/.+",
```